### ISSUE TITLE
Mood switches have stopped working

### ISSUE THREAD

**Issue by calston07** [soler2000/com.lightwaverf#39]

**Comment by calston07** [comment]
Since last weeks Homey firmware update Homey is no longer responding to any of my Mood switches (LW101). Is this just me or are others having issues?
Any thoughts on how to address this

Thanks

**Comment by CreativeDigital** [comment]
Yep same problem since the update, all my wireless switches LW205 have stopped working, I emailed Athom and they referred me to the app developer.

**Comment by calston07** [comment]
I received the same response. This site was the link that opened when I clicked report a problem. I’m presuming this is correct?

**Comment by CreativeDigital** [comment]
I think so.

**Comment by nigetech** [comment]
Bump! Any update on when this will likely be fixed? Cheers

**Comment by soler2000** [comment]
Sorry to say but I have too much on to maintain this.   If would be happy if someone would like to take this over and maintain this for the community.  This would be needed to support the lastest generation of LW hardware.   This app was started due to buying a homey and expecting it to be compatible with Lightwave,  not knowing that it was only a plan for official support and not a clear time line.

**Comment by nigetech** [comment]
Thanks for the response, completely understandable. Do you know what would have caused it to stop working suddenly? If there was no recent update rolled out which stopped it working we can pick it up with athom. Thanks

**Comment by soler2000** [comment]
It’s hard to say,  it’s still all working on my system and i’m running the same version as you.

**Comment by calston07** [comment]
Hey Nigetech did you hear anything back from the athom guys?  
Has anyone else managed to fix this?

**Comment by nigetech** [comment]
Heard nothing back from athom, tried on social media also. Seems to be affecting some and not others. Homey is currently unplugged and back to using domoticz. Shame but there seems to be a severe lack of support for it.

**Comment by calston07** [comment]
I contacted Athom again before Christmas and they said they were trying to contact the developer to discuss taking over control of the app so that they could support it going forward. Fingers crossed on that one. 
Very frustrating as Lighteaverf support was them main feature that lead me to purchase Homey

**Comment by nigetech** [comment]
Sounds more promising, same, majority of the house is LightwaveRF mood controlling hue/fibraro/lifx and other other stuff, so when everything stopped working overnight it was a nightmare!

**Comment by CreativeDigital** [comment]
I think it might be in Athom interests next time when making updates to communicate with developers of the apps, to make sure that code is written to a specified standard to ensure this doesn't happen again. It reflects very badly on the overall product.

**Comment by soler2000** [comment]
Nobody has contacted me about taking this over,  however I do give my permission if needed

**Comment by calston07** [comment]
Hi Soler2000,
 
A guy call Bram at Athom said that he had reached out to the developer before Christmas to ask if you were prepared to had the app over to them but they had had no response. 

I think we all really appreciate your time in developing the app and completely understand that you have limited time to maintain (especially when Athom change things without full consultation). But I think we would all also appreciate if there was any chance you could liase with Athom and pass over the app to them

Many thanks

Chris

**Comment by soler2000** [comment]
I have taken a look I the athom forum and there are no new messages but that’s the old one before they moved across.   I’ve created an account in the new forum but there does not seam to be a light wave thread.   

If your in contact with Bram point him here so I can hand over what ever is needed.

**Comment by calston07** [comment]
Hi

I’ve forwarded the details to Bram at Athom and they have responded to say they will reach out to you after the holidays

Thanks

Chris

**Comment by calston07** [comment]
Soler2000 did the Homey team ever contact you?  The last update from them 21st January they said they are still awaiting a response?

**Comment by soler2000** [comment]
Not that I am aware of.

**Comment by calston07** [comment]
Soler2000. For some reason the guys at Homey are finding it difficult to contact you despite me giving them the link to this forum. 
They have asked me if I can pass on their contact details in a hope you could reach out to them to help transfer the app to their control. Sorry for keeping pushing the agenda but I would be great full if you could contact Bram at their e-mail to support@athom.com. 
Many thanks for all your support

Chris

**Comment by nigetech** [comment]
They are aware now, after hassling them many times they commented back on their Instagram that they’re aware of it and are rebuilding the app.

Fingers crossed won’t take too long!
